src/penalty_parameter_functions.jl

```diff
@@ -114,7 +114,10 @@ function process_penalty_subproblem(r::ProportionalPenaltyParameter,
 
         if haskey(r.penalties, xhid)
             if !isapprox(r.penalties[xhid], r_value)
-                error("Penalty parameter must match across scenarios. Got $(r.penalties[xhid]) instead of $(penalty) for variable $(name(phd, xhid)). This error likely means the coefficients in the objective function do not match in the scenario subproblems.")
+                error("Penalty parameter must match across scenarios. Got $(r.penalties[xhid]) instead of $(penalty) for variable $(name(phd, xhid)) when processing penalty parameter for scenario $(scid). This error could mean that:
+                    1. The coefficients in the objective function do not match in the scenario subproblems
+                    2. The variable specific penalties were set in the penalty parameter before the call to `PH.solve`."
```

Comment:
The code should actually be able to handle being given the same penalty parameter for multiple solve commands without producing unusual results.

I'll reopen the issue and put this on there too.